packages/docusaurus/src/webpack/plugins/ChunkAssetPlugin.ts

```diff
@@ -25,16 +27,18 @@ class ChunkAssetPlugin {
           chunkNameToId[chunkName] = chunkId;
         });
         const buf = [source];
-        buf.push('');
-        buf.push('// function to get chunk assets');
+        buf.push('// function to get chunk asset');
         buf.push(
-          // If chunkName is passed, we convert it to chunk id
-          // Note that jsonpScriptSrc is an internal webpack function
-          `${
-            mainTemplate.requireFn
-          }.gca = function(chunkId) { chunkId = ${JSON.stringify(
+          // If chunkName is passed, we convert it to chunk asset url
+          // .p => public path url ("/" or "/baseUrl/")
+          // .u(chunkId) => chunk asset url ("assets/js/x63b64xd.contentHash.js")
+          // not sure where this is documented, but this link was helpful: https://programmer.help/blogs/5d68849083e1a.html
+          //
+          // Note: __webpack_require__.gca() is called in docusaurus.ts for prefetching
+          // Note: we previously used jsonpScriptSrc (Webpack 4)
+          `__webpack_require__.gca = function(chunkId) { chunkId = ${JSON.stringify(
             chunkNameToId,
-          )}[chunkId]||chunkId; return jsonpScriptSrc(chunkId); };`,
+          )}[chunkId]||chunkId; return __webpack_require__.p + __webpack_require__.u(chunkId); };`,
```

Comment:
@alexander-akait does it look like an equivalent code? `jsonpScriptSrc(chunkId)` == `__webpack_require__.p + __webpack_require__.u(chunkId);` ?